shared/types/user.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod'
+
+export const UserBaseDto = z.object({
+  id: z.string(),
+  username: z.string(),
+  displayname: z.string(),
+  views: z.number().nonnegative(),
+})
+
+export type UserBaseDto = z.infer<typeof UserBaseDto>
```

Comment:
Could we perhaps go with my earlier proposal of simply calling this a User, then alias in server as UserBaseDto as needed?

Reply:
https://github.com/opengovsg/askgovsg/pull/216/commits/487d91c2aa6ae2bd43e866ac8edc41c86b080448